platform/ios/src/MGLMapView.mm

```diff
@@ -702,7 +705,7 @@ - (void)createGLView
     _glView.delegate = self;
 
     CAEAGLLayer *eaglLayer = MGL_OBJC_DYNAMIC_CAST(_glView.layer, CAEAGLLayer);
-    eaglLayer.presentsWithTransaction = YES;
+    eaglLayer.presentsWithTransaction = NO;
```

Comment:
I think it’s safe to assume that there’s no difference between setting `presentsWithTransaction` to `NO` and not setting it at all, since it’s a property rather than an action method.